spec/media/media_spec.js

```diff
@@ -0,0 +1,137 @@
+describe('Media', function() {
+  var media, Media, Flag;
+  beforeEach(function() {
+    media = require('../../src/media/media');
+    Media = media.Media;
+    Flag = media.Flag;
+  });
+
+  afterEach(function() {
+    React.unmountComponentAtNode(root);
+  });
+
+  describe('Creates a basic media component', function() {
+    beforeEach(function() {
+      var image = (<img src="http://placehold.it/20x20"/>);
+      React.render((<Media className="media-class" leftImage={image}>fop</Media>), root);
+    });
+
+    it('creates a Media component', function() {
+      expect('.media .media-body').toContainText('fop');
+    });
+
+    it('passes the provided className onto the .media element', function() {
+      expect('.media').toHaveClass('media-class');
+    });
+  });
+
+  describe('when left image src is set', function() {
+    describe('when left image src is set', function() {
```

Comment:
maybe change second describe to 'when padding is normal' or something less confusing.